CONTRIBUTING.md

```diff
@@ -0,0 +1,245 @@
+Group](https://www.w3.org/community/solid/) (CG) to meet the needs of the [Solid
+Project](https://solidproject.org/).
+
+The TRs include specifications, use cases and requirements, best practices and
+guidelines, primers and notes about the Solid ecosystem.
+
+
+## Code of conduct
+
+Read the [Solid Code of
+Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md) and the
+[Positive Work Environment at W3C: Code of Ethics and Professional
+Conduct](https://www.w3.org/Consortium/cepc/) to keep our community approachable
+and respectable.
+
+
+## Contributions
+
+In order to be a substantive contributor to work items, you must be a member of
+the CG. It’s easy to [join the CG](https://www.w3.org/community/solid/join) if
+you’d like to contribute. People agree to the [W3C Community License Agreement
+(CLA)](http://www.w3.org/community/about/agreements/cla/) upon joining the CG.
+
+The works in this repository use the [MIT
+license](https://github.com/solid/specification/blob/main/LICENSE.md).
+
+We accept different types of contributions as described below.
+
+### Discussions
+
+If you'd like help troubleshooting a pull request (PR) you're working on, have a
+great new idea, or want to share implementation feedback, join us in [Solid
+specification chat](https://gitter.im/solid/specification).
+
+### Issues
+
+[Issues](https://docs.github.com/en/github/managing-your-work-on-github/about-issues)
+are used to track tasks that contributors can help with. If an issue has a
+triage label, we haven't reviewed it yet and you shouldn't begin work on it.
+
+If you've found something in the content or the website that should be updated,
+search open issues to see if someone else has reported the same thing. If it's
+something new, open an issue. We'll use the issue to have a conversation about
+the problem you want to fix.
+
+### Pull requests
+
+A [pull
+request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
+is a way to suggest changes in our repository.
+
+### Translations
+
+We aim to have our documents internationalized and available in multiple
+languages. The source content in this repository is written in English. We
+accept contributions to localize the English content.
+
+
+## Contributions at source repository
+
+Each CG work item listed at [Solid Technical
+Reports](https://solidproject.org/TR/) has its own repository and any issues
+and PRs should be made at their source.
+
+The persistent version of technical reports published in this repository (see
+[publishing technical reports](#publishing-technical-reports) are intended to
+be read-only snapshots of the CG work items.
+
+[Solid
+Panels](https://github.com/solid/specification/blob/main/README.md#solid-panels)
+focus on certain work streams, with an aim to propose technical reports for
+the Solid ecosystem.
+
+To build shared understanding, [user
+stories](https://github.com/solid/user-stories) can also be submitted.
```

Comment:
```suggestion
A low-barrier way to add concrete suggestions is to submit a [user
story](https://github.com/solid/user-stories). User stories are descriptions of desired features written in a special way, see the description there for details. User stories may then be used when formulating use cases and requirements.
```

Reply:
I decided I needed to figure out how to make concrete suggestions, so there you go!